Template/Tag/MatomoTag.php

```diff
@@ -51,7 +52,11 @@ public function getParameters()
         });
         $isEcommerceView = $this->makeSetting('isEcommerceView', false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
             $field->title = Piwik::translate('TagManager_MatomoTagEcommerceViewIsEcommerceView');
-            $field->description = Piwik::translate('TagManager_MatomoTagEcommerceViewIsEcommerceViewHelp');
+            $field->inlineHelp = '';
+            if (!Manager::getInstance()->isPluginActivated('Cloud')) {
```

Comment:
Let me know if there's a better way to determine if the environment is cloud or on-prem. But I tested locally with activated/deactivated Cloud plugin and it seems to work.